### ISSUE TITLE
Scale attribute on CSS files

### ISSUE THREAD

**Issue by chraebsli** [validator/validator#1091]
URL being validated or code to reproduce error:
on CSS files with a `scale` attribute it says that this attribute doesn't exist but it does

```
Error: scale: Property scale doesn't exist.
```

**Comment by cvrebert** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/scale

To be fair, it seems pretty new and only FF supports it currently.

**Comment by sideshowbarker** [comment]
Thanks for reporting this — fixed in the sources and pushed to https://validator.w3.org/nu/

**Comment by mig281** [comment]
Hi there. I definitely kept getting this error more recently and I had to switch from `scale: foo;` to `transform: scale(foo);` to keep the validator happy. This was my original usage:

```css
#foo, .bar {
  scale: 0.8;
}
```

However, this is the usage that the validator accepted:

```css
#foo, .bar {
  transform: scale(0.8);
}
```

The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/scale) do state:
>The scale CSS property allows you to specify scale transforms individually and independently of the transform property. This maps better to typical user interface usage, and saves having to remember the exact order of transform functions to specify in the transform value.

Would you be willing to look into a possible regression for this issue? My many thanks!

**Comment by GitThingsDan** [comment]
I checked my style.css and it has the same problem, but with the property 'rotate' ('scale' passed with no problem though).
![image](https://github.com/validator/validator/assets/143112461/4ddf08bf-a56d-4328-adae-667f53d44992)

**Comment by sideshowbarker** [comment]
@GitThingsDan Please raise a new separate issue — either in this issue tracker, or else, if you can reproduce this issue with the https://jigsaw.w3.org/css-validator/, the raise the issue in the https://github.com/w3c/css-validator/issues tracker.

And for the record here, the HTML checker does no longer report any error for the following:

```
#foo, .bar {
  scale: 0.8;
}
```
So the original issue reported here about the `scale` property has been resolved.

**Comment by Mikey670** [comment]
The real problem that I ran into with this one was when I wanted to animate scale and another transform property separately. EX:
`.box {
    scale: 1;
    rotation: 180deg;
    transition: scale 1sec, rotation 2sec;
}
.box:hover {
    scale: 2;
    rotation: -180deg;
}`

This is just an example, it is not actually what I was trying to do.
